src/nft/queries/genie/WalletAssetsFetcher.ts

```diff
@@ -38,10 +38,10 @@ export const fetchWalletAssets = async ({
     return {
       ...asset,
       collectionIsVerified: asset.asset_contract.isVerified,
-      lastPrice: asset.last_sale && formatEther(asset.last_sale.total_price),
+      lastPrice: asset.lastSale && formatEther(asset.lastSale.total_price),
```

Comment:
this new param is only on the v3 endpoint so in this fetcher we also need to update the url to be `REACT_APP_GENIE_V3_API_URL`